docs/api-docs/tax/tax-properties.md

```diff
@@ -1,14 +1,19 @@
 # Tax Properties for Products
 
-The Tax Properties API allows merchants to attach multiple tax codes to a given product. These tax codes allow merchants to use more than one tax provider; for example, a merchant might use different tax providers in different countries. They can also assist merchants who are transitioning between tax providers. 
+The Tax Properties API allows merchants to vary the tax info they send to tax providers. In some jurisdictions, tax rates for products vary by product composition. For example, correct taxation of alcohol requires info about alcohol percentage. Using tax properties, providers can return more accurate tax calculations. The examples in this guide expand on this use case. 
 
-In addition, the tax rates for some products vary by product composition. For example, in some jurisdictions, correct taxation of alcohol and fuel requires more than one tax code. To provide accurate calculations, tax providers need to know about both the percentage of alcohol by volume **and** the volume of alcohol sold. With tax properties, merchants can send tax providers more than one code per product. Providers can return more accurate tax calculations. The examples in this guide expand on this use case.
+Tax providers may sometimes need multiple inputs that vary between products. For example, the alcohol percentage and the volume sold affect alcohol taxes. As shown in this guide, merchants can provide both pieces of info to tax providers.    
```

Comment:
```suggestion
Tax providers may sometimes need multiple inputs that can vary between different types of product. For example, for products with alcohol, the alcohol percentage and the volume sold affect alcohol taxes. As shown in this guide, merchants can provide both pieces of information to tax providers.
```

If you think it makes sense to keep, sounds fine to me @bc-andreadao 👍 

🍹 I've tried to rejig this a little to speak to different product types. So, for example, a store may have alcohol items but they may also have some other type of product (that would benefit from tax properties also) - they can introduce tax properties to support both product types. Maybe this would be good to expand further? Unsure.

Reply:
sounds good to me!